### PR TITLE
Pegasus M13 experiments runner

### PR DESCRIPTION
**Pull request by spigo900**

### CODE REVIEW

**Comment by spigo900** [comment]
This should be complete, but I think it makes sense to hold off on merging it until #949 is merged since that makes changes to the M13 runner as well.

File: adam/experiment/__init__.py [hunk 1]
```diff
@@ -143,6 +169,19 @@ def execute_experiment(
...
+            "Cannot both resume from latest logged state and load a specified state file."
+        )
+
+    if resume_from_latest_logged_state and learner_logging_path is None:
+        raise RuntimeError(
+            "Need state logging path to be able to resume from latest learner state."
+        )
+
+    if resume_from_latest_logged_state and starting_point != -1:
+        raise RuntimeError("Starting point should not be specified when")
```
Comment (LichMaster98) [inline adam/experiment/__init__.py:183]: This error message looks incomplete.

**Review by LichMaster98 — CHANGES_REQUESTED** [review]

File: adam/experiment/__init__.py (hunk 1 above)
Comment (spigo900) [reply adam/experiment/__init__.py:183]: And it is. :)

Thanks for catching this.

Comment (spigo900) [reply adam/experiment/__init__.py]: This should be fixed now.

**Review by LichMaster98 — APPROVED** [review]

**Pull request merged**